util/all_tests.go

```diff
@@ -411,6 +419,11 @@ func main() {
 	go func() {
 		for _, baseTest := range testCases {
 			test := test{Test: baseTest}
+			if *useValgrind {
+				if test.SkipValgrind {
+					continue
+				}
+			}
```

Comment:
Nit: need new if block? use `&&`?

Reply:
Style question. I'll leave it as it is for now.